.github/workflows/awie.yml

```
@@ -42,8 +56,8 @@ jobs:
       veracode_srcclr_token: ${{ secrets.VERACODE_SRCCLR_TOKEN }}
 
   package:
-    needs: [get-version]
-    if: ${{ needs.get-version.outputs.stability != 'stable' }}
+    needs: [get-version, check-version]
```

```suggestion
    needs: [check-version]
```
As `get-version` is already required before executing `check-version`

---

since packaging job relies on the get-version output, linter/gh may complain about it as being a bad practice, we should leave both get-version and check-version